rustls/src/key.rs

```diff
@@ -15,6 +15,10 @@ pub struct PrivateKey(pub Vec<u8>);
 /// The certificate must be DER-encoded X.509.
 ///
 /// The `rustls-pemfile` crate can be used to parse a PEM file.
+///
+/// ## Note
```

Comment:
Style nit: I think convention is to have an empty line after a heading.

Reply:
Good catch, I've added the empty line.